src/Components/Radio.js

```diff
@@ -7,19 +7,19 @@ const Radio = () => {
     <div className="flex-1">
       <form >
         <span className="p-4">
-              <input className="form-radio text-indigo-600" type="radio" value="Listmonk" id="Listmonk" name="mail" onChange={()=>{setservice('Listmonk')}} checked={service==='Listmonk'}/>
+              <input className="m-0.5" type="radio" value="Listmonk" id="Listmonk" name="mail" onChange={()=>{setservice('Listmonk')}} checked={service==='Listmonk'}/>
```

Comment:
The spacing between the radio button and the label is negligible. A little more spacing between them would make the UI look even better :)